templates/bmc.yml.erb

```diff
@@ -5,3 +5,11 @@
 # - freeipmi / ipmitool - requires the appropriate package installed, and the rubyipmi gem
 # - shell - for local reboot control (requires sudo access to /sbin/shutdown for the proxy user)
 :bmc_default_provider: <%= scope.lookupvar("foreman_proxy::bmc_default_provider") %>
+<% if scope.lookupvar("foreman_proxy::bmc_default_provider") == 'ssh' -%>
+:bmc_ssh_user: <%= scope.lookupvar("foreman_proxy::bmc_ssh_user") %>
+:bmc_ssh_key: <%= scope.lookupvar("foreman_proxy::bmc_ssh_key") %>
+:bmc_ssh_powerstatus: <%= scope.lookupvar("foreman_proxy::bmc_ssh_powerstatus") %>
```

Comment:
This should be a string but currently it's printed as a boolean. In fact, I think it'd be good to quote all values as strings.

Reply:
I followed the other templates in this project which also do not quote strings. I could quote them for sure but it would be inconsistent. Example:

```
#templates/puppetca.yml.erb
<% if scope.lookupvar("foreman_proxy::puppetca_modular") -%>
:use_provider: <%= scope.lookupvar("foreman_proxy::puppetca_provider") %>
<% else -%>
:autosignfile: <%= scope.lookupvar("foreman_proxy::autosignfile") %>
<% end -%>
```

Reply:
I looked at https://github.com/theforeman/smart-proxy/blob/e5e524ba2f32ca37111a89f74fc0edb0ef51c89d/config/settings.d/bmc.yml.example#L16-L19 and these must be quoted I think. The `true` and `false` refer to `/bin/true` and `/bin/false`. If they're not quoted, they become booleans which could behave different.

Reply:
Of course, that makes sense. I did another commit where all the commands are quoted.